app/views/users/new.html.erb

```diff
@@ -1,4 +1,20 @@
 <% provide :title, t("users.signup.header_title") %>
 
 <h1><%= t "users.signup.header_title" %></h1>
-<p><%= t ".introduction" %></p>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for User.new do |f| %>
+      <%= render "shared/error_messages" %>
+      <%= f.label :name, t(".name_title") %>
```

Comment:
tìm hiểu thay đổi tên attributes trong i18n, thay thế việc define trực tiếp ở view
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models
```suggestion
      <%= f.label :name %>
```